### PR TITLE
Emoji Maker Project using Vanilla JS 

### PR DESCRIPTION
**Pull request by SohanRC**
# Pull Request for WebMasterLog 💡

## Issue Title: Emoji Maker Project using Vanilla JS 

- **Name of the project** : Emoji Maker
- **Category:** Vanilla-JS
- **Name:** Sohan RoyChowdhury
- **GitHub ID:** https://github.com/SohanRC
- **Idenitfy yourself:** JWOC 2024

Closes: 
#66 

### Describe the add-ons or changes you've made 📃

This Project is an Emoji Maker project made using HTML , CSS & JS. This project allows users to make their custom emojis according to their choices . It also enables them to download the emojis after they are made.

## Type of change ☑️

What sort of change have you made:

- [X] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Tested In on local System and It worked perfectly !

## Checklist: ☑️

- [X] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [X] This PR does not contain plagiarized content.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

Screenshot 2024-02-11 130827

### CODE REVIEW

**Comment by SohanRC** [comment]
@Avdhesh-Varshney Please check this PR !!

File: Vanilla-JS-Projects/Intermediate/Emoji-Maker/Readme.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Emoji-Maker/Readme.md:1]: use the updated README TEMPLATE.

File: Vanilla-JS-Projects/Intermediate/Emoji-Maker/eye-0.svg
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Emoji-Maker/eye-0.svg:1]: Create a separate directory for these images inside Emoji-Maker and keep all the required assets in a separate directory. Only keep the screenshot of the working website in .webp format just inside the project directory.

File: Vanilla-JS-Projects/Intermediate/Emoji-Maker/index.html
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Emoji-Maker/index.html:1]: After shifting link those images properly and check whether the website is working or not?

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@SohanRC do these changes, again mentioned to me.

**Comment by SohanRC** [comment]
@Avdhesh-Varshney I have made the required changes. Please review it now !

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Comment by Avdhesh-Varshney** [comment]
@SohanRC properly linked your issue so that it will automatically closed on merging the pr.

**Comment by SohanRC** [comment]
> @SohanRC properly linked your issue so that it will automatically closed on merging the pr.

Thank You!!

**Pull request merged**